packages/component/src/SendBox/SuggestedActions.js

```diff
@@ -40,6 +41,7 @@ const connectSuggestedActions = (...selectors) =>
   );
 
 const SuggestedActions = ({ className, suggestedActions = [] }) => {
+  const [direction] = useDirection();
```

Comment:
Sort. I believe `[direction]` should be ASCII-sorted below `[{ suggestedActionLayout, ...}]`.